Microsoft.Packaging.Tools/tasks/targets/Microsoft.Packaging.Tools.targets

```diff
@@ -38,21 +30,8 @@
     </Otherwise>
   </Choose>
 
-  <Target Name="_GetLockFileAssemblies"
-          DependsOnTargets="ComputePrivateAssetsPackageReferences;
-                            _DefaultMicrosoftNETPlatformLibrary">
-    <!-- Essentially a copy of the SDKs RunResolvePublishAssemblies.
-         We need to find all the files that will be loaded from deps for conflict resolution.-->
-    <ResolvePublishAssemblies ProjectPath="$(MSBuildProjectFullPath)"
-                              AssetsFilePath="$(ProjectAssetsFile)"
-                              TargetFramework="$(TargetFrameworkMoniker)"
-                              RuntimeIdentifier="$(RuntimeIdentifier)"
-                              PlatformLibraryName="$(MicrosoftNETPlatformLibrary)"
-                              PrivateAssetsPackageReferences="@(PrivateAssetsPackageReference)">
-
-      <Output TaskParameter="AssembliesToPublish" ItemName="_LockFileAssemblies" />
-
-    </ResolvePublishAssemblies>
+  <Target Name="PackagingToolsInsufficientSDKVersion" Condition="'$(PackagingToolsInsufficientSDKVersion)' != 'false'" AfterTargets="$(PackagingToolsInsufficientSDKVersionAfter)">
+    <Error Text="The version of Microsoft.NET.Sdk used by this project is insufficient to support this package.  This error may be suppressed by setting PackagingToolsInsufficientSDKVersion=false" />
```

Comment:
Yes, that's my understanding too.  https://blogs.msdn.microsoft.com/visualstudio/2017/05/10/update-to-visual-studio-2017-and-next-preview/

> Today, we are also making available Visual Studio 2017 version 15.3 preview.

Your error message looks good to me.